docs/pages/versions/unversioned/sdk/facebook.md

```diff
@@ -47,16 +47,58 @@ You may have to switch the app from 'development mode' to 'public mode' on the F
 import * as Facebook from 'expo-facebook';
 ```
 
-### `Facebook.initializeAsync(appId: string | undefined, appName: string | undefined): Promise<void>`
+### `Facebook.initializeAsync(options: InitOptions): Promise<void>`
```

Comment:
I thought we decided against using the SDK name in the type:
```diff
import * as Facebook from 'expo-facebook';

- Facebook.FacebookInitializationOptions
+ Facebook.InitOptions
```

Reply:
on internal types it doesn't really matter but i thought we agreed that we would use the name in exported types

Reply:
The rationale for choosing verbosity in exported types is that it is usually a good heuristic for code readability because the context is in the name. `InitOptions` in the middle of a file carries less context than `FacebookInitializationOptions`, especially when other packages might have their own `InitOptions` types.

Another way to address this is to write out: `import * as Facebook from 'expo-facebook'; Facebook.InitOptions` but AFAIK bundler tools these days interpret `import *` as "import all exports" instead of "analyze the rest of the file and import what's actually potentially used".

So, the pit of success looks like: `import { FacebookInitializationOptions } from 'expo-facebook'`: named imports help with tree shaking (dead-export elimination) and the named import has a name that is unlikely to collide with imports from other modules and communicates more of its context in its name.

For those who want just `InitOptions`, `import { FacebookInitializationOptions as InitOptions } from 'expo-facebook'` is the way to go (or to write a small intermediate module that imports and re-exports everything from expo-facebook).